src/tests/unit/__vitest_data__/instant-loan.ts

```diff
@@ -83,7 +83,7 @@ const branchHoldings: HoldingsV3[] = [
         periodical: undefined,
         materialGroup: {
           name: "standard",
-          description: "I am supposed to be matched, seriously!"
```

Comment:
Are you sure that the text should be changed here?
I do not remember clearly. But the word `seriously` was probably added to make it distinct/stick out?

Reply:
The new change dos that it is only the exact string that causes it to match.